ignite/cmd/chain.go

```diff
@@ -18,13 +18,17 @@ import (
 	"github.com/ignite/cli/ignite/pkg/cosmosgen"
 	"github.com/ignite/cli/ignite/pkg/goanalysis"
 	"github.com/ignite/cli/ignite/pkg/xast"
+	"github.com/ignite/cli/ignite/services/chain"
 )
 
 const (
-	msgMigration       = "Migrating blockchain config file from v%d to v%d..."
-	msgMigrationPrefix = "Your blockchain config version is v%d and the latest is v%d."
-	msgMigrationPrompt = "Would you like to upgrade your config file to v%d"
-	toolsFile          = "tools/tools.go"
+	msgMigration            = "Migrating blockchain config file from v%d to v%d..."
+	msgMigrationPrefix      = "Your blockchain config version is v%d and the latest is v%d."
+	msgMigrationPrompt      = "Would you like to upgrade your config file to v%d"
+	msgMigrationBuf         = "Now ignite supports the `buf.build` (https://buf.build) registry to manage the protobuf dependencies. The embed protoc binary was deprecated and, your blockchain is still using it. Would you like to upgrade and add the `buf.build` config files to `proto/` folder"
```

Comment:
@ccomben can you please help improving this text?

Reply:
Maybe?

```suggestion
	msgMigrationBuf         = "Buf (https://buf.build) is required for proto code generation. Would you like to enable it?"
```

Users would get the following message if `n` is selected:

```
code generation using protoc is only supported by Ignite CLI v0.26.1 or older
```

A change for this message is proposed in another comment.